packages/components/pie-modal/package.json

```diff
@@ -39,7 +39,7 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@custom-elements-manifest/analyzer": "0.9.0",
-    "@justeat/pie-design-tokens": "7.4.0",
+    "@justeat/pie-design-tokens": "7.6.1",
```

Comment:
Not something to fix in this PR, but any idea why the design tokens are a devDependency but only in `pie-modal`?

Reply:
for this: https://github.com/justeattakeaway/pie/blob/main/packages/components/pie-modal/src/modal.scss#L126